### DISCUSSION TITLE
Making the Pattern Data Value display in hexadecimal

### DISCUSSION THREAD

**Discussion by Lexden12** [WerWolv/ImHex#1914] [Get Help] [answered]
Hello, sorry if this is supposed to be easy to find, but after searching through documentation and online, I can't seem to find an answer...
I want the Pattern Data Value to display in hexadecimal rather than decimal:
![image](https://github.com/user-attachments/assets/c91e68d8-ae15-4fa0-8012-b179a7d703d0)
I saw in the documentation that this can clearly be done, but I could not find any place documented how to switch to this mode:
![image](https://github.com/user-attachments/assets/e6dbdc07-cba6-4a8c-9d95-0f66ecb27c78)

**Comment by WerWolv** [comment, marked as answer]
Hey the easiest way to do it is by using the `type.base` library.

```cpp
import type.base;

type::Hex<u32> hexValue @ 0x00;
```

Can be shortened down with a `using` declaration of course, if you prefer that.

**Reply by Lexden12** [reply]
Thank you, I very much appreciate the help!
